howto/manage/web-dashboard.md

```diff
@@ -1,29 +1,9 @@
-The Anbox Cloud Dashboard offers a web GUI from where developers can create, manage, and even stream applications from their web browser.
-The dashboard is useful if you are new to Anbox Cloud or want a simple management interface.
+The Anbox Cloud Dashboard offers a web GUI that developers can use to create, manage and stream applications from their web browser. You can use the pre-installed dashboard almost immediately after deploying Anbox Cloud.
```

Comment:
Is it intentional to categorise users of the dashboard as developers here ?

Reply:
I have now made it more generic. Calling out target audience is good for specificity but in this instance, I think it is better to start with non-specificity.